src/test/scala/com/microsoft/hyperspace/index/IndexCacheTest.scala

```diff
@@ -166,18 +165,30 @@ class IndexCacheTest extends SparkFunSuite with SparkInvolvedSuite {
   }
 }
 
+class MockClock(startTime: Long = 1L) extends Clock {
+  private var time: Long = startTime
+
+  override def getTime: Long = time
+
+  def moveTime(value: Long): Unit = {
```

Comment:
nit: `def advance`?

Reply:
Also, since we don't use `startTime`, we can just simplify as:
```scala
class MockClock(private var curTime: Long = 0L) extends Clock {
  override def getTime: Long = curTime

  def advance(value: Long): Unit = {
    curTime = curTime + value
  }
}
```